website/docs/r/cloudfront_distribution.html.markdown

```diff
@@ -251,7 +251,7 @@ of several sub-resources - these resources are laid out below.
     requests, the Id of the AWS WAF web ACL that is associated with the
     distribution. The WAF Web ACL must exist in the WAF Global (CloudFront)
     region and the credentials configuring this argument must have
-    `waf:GetWebACL` permissions assigned.
+    `waf:GetWebACL` permissions assigned. Provide the ARN if using wafv2.
```

Comment:
Nit: let's bring the reference to wafv2 into an `if` condition to follow the 1st sentence's pattern and specify which ARN we're referring to just in case some ambiguity is introduced e.g. 
`"If using WAFv2, provide the ARN of the web ACL."`

Reply:
Sure, I just made the update.